components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -498,6 +498,9 @@ protected void processAuthenticationResponse(HttpServletRequest request,
                                                  HttpServletResponse response, AuthenticationContext context)
             throws AuthenticationFailedException {
 
+        context.setProperty("captchaParams",
```

Comment:
shall we make captchaParams a constant

Reply:
Addressed in 737888e93ef15c373e5e0fe56126e3771e5be7da